### PR TITLE
create canned markets then zeroX orders

### PR DESCRIPTION
**Pull request by bthaile**
add convenience flash script that uses existing canned markets creation and scripts to create zeroX orders for each market type.

lowercase existing zerox order script names for ease of typing.

`create-canned-markets-with-orders` will create all markets first then create all zeroX orders

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://dashboard.featurepeek.com). 🌈
Your deployment will be kept up-to-date with this pull request's latest changes.

**https://peek.run/r9zhjwp**

-------

*Please read [our docs](https://docs.featurepeek.com/configuring-peek-yml/) for more configuration details.*

File: packages/augur-tools/src/flash/scripts.ts [hunk 1]
```diff
@@ -733,12 +792,47 @@ export function addScripts(flash: FlashSession) {
...
     ],
     async call(this: FlashSession, args: FlashArguments) {
+      const endpoint = 'ws://localhost:60557';
       const address = args.userAccount as string;
-      const user = await this.ensureUser(null, null, true, address);
-      const type =
-        String(args.orderType).toLowerCase() === 'bid' || 'buy' ? 0 : 1;
+      const isZeroX = args.zerox as boolean;
+      const fillOrder = args.fillOrder as boolean;
+      let user = null;
```
Comment (robert-e-davidson3) [inline packages/augur-tools/src/flash/scripts.ts]:
```suggestion
      let user: ContractAPI;
```

File: packages/augur-tools/src/flash/scripts.ts [hunk 2]
```diff
@@ -749,105 +843,109 @@ export function addScripts(flash: FlashSession) {
...
-        onChainShares,
-        onChainPrice,
-        new BigNumber(String(args.outcome)),
-        nullOrderId,
-        nullOrderId,
-        tradegroupId
-      );
-
+      const tradeGroupId = stringTo32ByteHex('tradegroupId');
+      let result = null;
```
Comment (robert-e-davidson3) [inline packages/augur-tools/src/flash/scripts.ts:849]: This isn't used except to be printed later, which will always be null.

File: packages/augur-tools/src/flash/scripts.ts [hunk 3]
```diff
@@ -749,105 +843,109 @@ export function addScripts(flash: FlashSession) {
...
+          onChainPrice,
+          new BigNumber(String(args.outcome)),
+          nullOrderId,
+          nullOrderId,
+          tradeGroupId
+        );
+      }
       this.log(`place order ${result}`);
+      // something is hanging. no missings await that I can see.
+      process.exit();
```
Comment (robert-e-davidson3) [inline packages/augur-tools/src/flash/scripts.ts]: Maybe put this in `cli.ts`? I'm thinking like this:
```
try {
    if (args.mode === 'interactive') {
      const vorpal = makeVorpalCLI(flash);
      flash.log = vorpal.log.bind(vorpal);
      vorpal.show();
    } else if (args.network === 'none') {
      flash.call(args.command, args).catch(console.error);
    } else {
      flash.network = NetworkConfiguration.create(args.network);
      flash.provider = flash.makeProvider(flash.network);
      flash.getNetworkId(flash.provider).then((networkId) => {
        flash.contractAddresses = Addresses[networkId];
        return flash.call(args.command, args);
      }).catch(console.error);
    }
  } finally {
    process.exit()
  }
```

Comment (robert-e-davidson3) [reply packages/augur-tools/src/flash/scripts.ts]: Oh that wouldn't work. Nevermind.

**Review by robert-e-davidson3 — APPROVED** [review]

**Pull request merged**